ci-operator/step-registry/aws/provision/kms-key/aws-provision-kms-key-commands.sh

```diff
@@ -14,7 +14,7 @@ trap 'if [[ "$?" == 0 ]]; then EXIT_CODE=0; fi; echo "${EXIT_CODE}" > "${SHARED_
 
 export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
 
-REGION="${LEASED_RESOURCE}"
+REGION=${REGION:-$LEASED_RESOURCE}
```

Comment:
@yunjiang29 FYI, I have to change it in this way because we must make sure the kms is in the same region with the hosted cluster